FFMediaToolkit/src/Decoding/StreamInfo.cs

```diff
@@ -100,6 +104,27 @@ internal unsafe StreamInfo(AVStream* stream)
         /// </summary>
         public ReadOnlyDictionary<string, string> Metadata { get; }
 
+        /// <summary>
+        /// Gets the stream chapters.
+        /// </summary>
+        public StreamChapter[] Chapters { get; }
+
         private static string FormatCodecId(AVCodecID id) => id.ToString().Substring(12).ToLower();
+
+        private static unsafe StreamChapter[] ParseChapters(InputContainer container)
+        {
+            var streamChapters = new StreamChapter[container.Pointer->nb_chapters];
+
+            for (var i = 0; i < container.Pointer->nb_chapters; i++)
+            {
+                var chapter = container.Pointer->chapters[i];
+                var meta = chapter->metadata;
+                var startTimespan = TimeSpan.FromTicks(chapter->start / 100);
```

Comment:
This will only work if the time base = 0.001. Instead, use the `chapter->time_base` value.
```suggestion
                var startTimespan = chapter->start.ToTimeSpan(chapter->time_base);
```